oci/private/image.bzl

```diff
@@ -166,6 +166,8 @@ def _oci_image_impl(ctx):
         # reuse given base image
         args.add(ctx.file.base.path, format = "--from=%s")
         inputs.append(ctx.file.base)
```

Comment:
did you mean to only call `inputs.append(ctx.file.base)` if `not use_symlinks`?

Reply:
No, base has to still stay as input because we read some files from there, but when symlinks are enabled we have to also add base into transitive inputs for downstream targets, if not symlinks will be dangling.